app/javascript/emoji_picker/EmojiPicker.vue

```diff
@@ -66,6 +62,8 @@ const { highlightedSearchable, onArrowDown, onArrowUp, topRankedMatches } =
     },
   });
 
+const googleLoginOrigin = window.location.href;
```

Comment:
(I'd have liked to simply inline this, but trying to do so was generating a TypeScript complaint.)